BuildAddOn.py

```diff
@@ -294,35 +311,45 @@ def GetDevKitVersion (args, devKitData, version, platformName):
 
 
 # Zip packages
-def PackageAddOns (args, devKitData, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
+def PackageAddOns (args, devKitData, addOnName, buildConfigList, acVersionList, languageList, buildFolder, packageRootFolder):
+    platformName = GetPlatformName ()
     Check7ZInstallation ()
 
     for version in acVersionList:
-        for languageCode in languageList:
-            CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'RelWithDebInfo', languageCode)
+        versionAndBuildNum = GetDevKitVersion (args, devKitData, version, platformName)
 
-            versionAndBuildNum = GetDevKitVersion (args, devKitData, version, platformName)
-            subprocess.call ([
-                '7z', 'a',
-                str (packageRootFolder.parent / version / f'{addOnName}-{versionAndBuildNum}_{platformName}_{languageCode}.zip'),
-                str (packageRootFolder / version / languageCode / '*')
-            ])
+        for languageCode in languageList:
+            for config in buildConfigList:
+                CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, config, languageCode)
+
+                if args.quiet:
+                    subprocess.call ([

```

Comment:
This is duplicated logic, I would create a list that represent the parts of the commands, like at other places.

Reply:
On windows, suppressing the output of a command is different in powershell and cmd. This approach is universal on all platforms.

Reply:
I mean the pattern that we use at other places:
```python
    params = [...]
    if quiet:
        subprocess.call (params, stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT)
    else:
        subprocess.call (params)
```

Reply:
Would be even better to create a `CallCommand` function with a `quiet` parameter so we don't have to duplicate `, stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT` everywhere.

Reply:
Fixed